src/main/java/it/gov/pagopa/spontaneouspayment/service/EnrollmentsService.java

```diff
@@ -0,0 +1,165 @@
+package it.gov.pagopa.spontaneouspayment.service;
+
+import java.util.stream.StreamSupport;
+
+import javax.validation.constraints.NotBlank;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.azure.cosmos.models.PartitionKey;
+
+import it.gov.pagopa.spontaneouspayment.entity.Organization;
+import it.gov.pagopa.spontaneouspayment.entity.ServiceRef;
+import it.gov.pagopa.spontaneouspayment.exception.AppError;
+import it.gov.pagopa.spontaneouspayment.exception.AppException;
+import it.gov.pagopa.spontaneouspayment.model.EnrollmentModel;
+import it.gov.pagopa.spontaneouspayment.model.OrganizationModel;
+import it.gov.pagopa.spontaneouspayment.repository.OrganizationRepository;
+import it.gov.pagopa.spontaneouspayment.repository.ServiceRepository;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@Service
+@AllArgsConstructor
+@NoArgsConstructor
+public class EnrollmentsService {
+
+    @Autowired
+    private OrganizationRepository orgRepository;
+    
+    @Autowired
+    private ServiceRepository serviceRepository;
+
+  
+    public Organization createEC(Organization orgEntity) {
+    	Iterable<it.gov.pagopa.spontaneouspayment.entity.Service> allServices = serviceRepository.findAll();
+    	
+    	// check if all services the EC wants to enroll are configured in the database
+    	for (ServiceRef servRef : orgEntity.getEnrollments()) {
+    		boolean exist = StreamSupport.stream(allServices.spliterator(), false).anyMatch(s -> s.getId().equals(servRef.getServiceId()));
+    		if (!exist) {
```

Comment:
use stream method `.isPresent()` instead.

Reply:
I didn't understand what you mean exactly